en-au/index.html

```diff
@@ -227,15 +227,15 @@ <h3 class="item-title">Truly decentralized.</h3>
                     <h3 class="item-title">More secure.</h3>
 
                     <p>
-					DigiByte uses 5 cryptographic algorithms and <a href="https://www.dgbwiki.com/index.php?title=DigiShield" target="_blank">real time difficulty adjustment</a> to prevent malicious mining centralization and hash power fluctuation. One of which is Odocrypt that changes itself every 10 days for ASIC resistance.
+					DigiByte uses 5 cryptographic algorithms and <a href="https://www.dgbwiki.com/index.php?title=DigiShield" target="_blank">real time difficulty adjustment</a> to prevent malicious mining centralization and hash power fluctuation. One of which is Odocrypt that changes itself every 10 days.
                     </p>
                 </div>
 
                 <div class="about-desc__slide">
                     <h3 class="item-title">Much faster.</h3>
 
                     <p>
-                    DigiByte blocks occur every 15 seconds which is 40x faster than Bitcoin and 10x faster than Litecoin. Early <a href="https://wikipedia.org/wiki/SegWit" target="_blank">SegWit</a> implementation and blockchain rigidity enable up to <a href="https://www.dgbwiki.com/index.php?title=DigiSpeed" target="_blank">1066</a> on-chain transactions per second with negligible fees.
+                    DigiByte blocks occur every 15 seconds which is 40x faster than Bitcoin and 10x faster than Litecoin. Early <a href="https://wikipedia.org/wiki/SegWit" target="_blank">SegWit</a> enables up to <a href="https://www.dgbwiki.com/index.php?title=DigiSpeed" target="_blank">4x</a> amount of transactions per second with negligible fees.
```

Comment:
"Early SegWit" sounds a bit clunky. I say let's just remove "Early" all together, because that fact doesn't really belong in that section in the first place. Let's make it "SegWit implementation enables up to..."

- For all English versions.

Reply:
removed the word "Early"
 see commit e838b1ae7038367eebf5cc6b1bc41c45937daa57